src/GafferImage/Merge.cpp

```diff
@@ -80,7 +82,7 @@ Merge::Merge( const std::string &name )
 			Plug::In,	// direction
 			Add,		// default
 			Add,		// min
-			Under		// max
+			Max			// the maximum value in the enum, which just happens to currently be named "Max"
```

Comment:
The rule to follow here is "tabs for indentation, spaces for alignment". So we should be using tabs up to "Max", and then spaces up to "//", and doing the equivalent on all the other lines. That's the only way to allow someone else to use different tab widths from you, but not mess up subsequent alignment. If you google "tabs for indentation, spaces for alignment", you'll get a few decent explanations for this, along with a fair amount of tabs vs spaces ranting.